components/dialpad/package.json

```diff
@@ -1,17 +1,28 @@
 {
   "name": "@pipedream/dialpad",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
I think you'll need to remake the package-lock.json file with the updated version.
npm i --package-lock-only

Reply:
okay